pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -433,12 +457,6 @@ func buildGatewayListenerTLSContext(
 	ctx := &tls.DownstreamTlsContext{
 		CommonTlsContext: &tls.CommonTlsContext{
 			AlpnProtocols: util.ALPNHttp,
-			TlsParams: &tls.TlsParameters{
-				TlsMinimumProtocolVersion: convertTLSProtocol(server.Tls.MinProtocolVersion, tls_features.TLSMinProtocolVersion.Get()),
```

Comment:
this is removing an OSSM feature (setting TLS defaults through env vars), we need this

Reply:
Reverted, Thanks.